### PR TITLE
My Jetpack: tweak dims of the Product card status

### PR DESCRIPTION
**Pull request by retrofox**
It tweaks the status component of the Product card, aligning it with the action button:

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: tweak dims of the Product card status

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Go to My jetpack dashboard

before | after
-------|------
image | image

*

### CODE REVIEW

File: projects/packages/my-jetpack/_inc/components/product-card/style.module.scss [hunk 1]
```diff
@@ -67,6 +67,8 @@
 .status {
   margin-left: var(--product-card-margin-base);
   white-space: nowrap;
+  height: 36px;
+  line-height: 36px;
```
Comment (renatoagds) [inline projects/packages/my-jetpack/_inc/components/product-card/style.module.scss:71]: Use the var here: var(--product-card-actions-size)

**Review by renatoagds — CHANGES_REQUESTED** [review]

File: projects/packages/my-jetpack/_inc/components/product-card/style.module.scss (hunk 1 above)
Comment (retrofox) [reply projects/packages/my-jetpack/_inc/components/product-card/style.module.scss]: addressed here ec654f582eed052d85cf8a8a886d83a87c6d3803

File: projects/packages/my-jetpack/_inc/components/product-card/style.module.scss [hunk 2]
```diff
@@ -67,6 +68,8 @@
 .status {
   margin-left: var(--product-card-margin-base);
   white-space: nowrap;
+  height: --product-card-actions-size;
+  line-height: --product-card-actions-size;
```
Comment (renatoagds) [inline projects/packages/my-jetpack/_inc/components/product-card/style.module.scss:72]: You need to put it inside `var()` ...

```css
var(--product-card-actions-size)
```

**Review by renatoagds — CHANGES_REQUESTED** [review]

File: projects/packages/my-jetpack/_inc/components/product-card/style.module.scss [hunk 3]
```diff
@@ -15,6 +15,7 @@
   --status-inactive: #646970;
   --status-error: #B32D2E;
   --status-plugin_absent: #C3C4C7;
+  --status-label-height: 36px;
```
Comment (renatoagds) [inline projects/packages/my-jetpack/_inc/components/product-card/style.module.scss:18]: Looks like this one is not being used. Can we remove it?

**Review by renatoagds — CHANGES_REQUESTED** [review]

**Comment by renatoagds** [comment]
Sorry for the long `requested changes`.

File: projects/packages/my-jetpack/_inc/components/product-card/style.module.scss (hunk 3 above)
Comment (retrofox) [reply projects/packages/my-jetpack/_inc/components/product-card/style.module.scss]: addressed here https://github.com/Automattic/jetpack/pull/22561/commits/43e936975c2fb083eb46b0f91c01604161190e34

**Review by renatoagds — APPROVED** [review]

**Pull request merged**